### DISCUSSION TITLE
issue during installation process

### DISCUSSION THREAD

**Discussion by duemme** [Homebrew/discussions#782] [Getting started]
I was installing Homebrew using this script (/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)") and I lost internet connection.
If I try to use the script again I get this error:

==> Downloading and installing Homebrew...
HEAD is now at d6cdc884d Merge pull request #10603 from Homebrew/sorbet-files-update
error: Not a valid ref: refs/remotes/origin/master
fatal: ambiguous argument 'refs/remotes/origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]' 

can someone help me fix it?
thanks

**Comment by gromgit** [comment]
Looks like you lost internet during the Homebrew software cloning process, so the easiest way to recover from this is to `rm -fr /usr/local/Homebrew`, then run the installer again.

**Reply by duemme** [reply]
thanks!

**Reply by kingofice007** [reply]
thanks!

**Reply by Meredlyo** [reply]
do i close the terminal and run this command from start?

**Comment by shaik-razor** [comment]
first run
`sudo rm -rf /opt/homebrew `
then run 
 `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
